src/engine/TextIndexScanForWord.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <string>
+
+#include "./Operation.h"
+
+// This operation filters all entries of a text-variable out that do not contain
+// a certain word or a certain prefix.
```

Comment:
This comment must be improved (similarly comment the other scan):
Something like "This operation retrieves all text records from the fulltext index that contain a certain word or prefix".